app/screens/home/DetailsScreen.tsx

```diff
@@ -239,22 +241,30 @@ const DetailsScreen = ({ route, navigation }: any): any => {
           if (articleData && articleData.length > 0) {
             setDetailDataToUse(articleData[0]);
             if (fromScreen === 'Activities' || fromScreen === 'FirebaseActivities' || fromScreen === 'MileStoneActivity' || fromScreen === 'HomeAct' || fromScreen === 'FavActivities') {
-              analytics().logEvent(GAME_CATEGORY_SELECTED + "_" + articleData[0]?.activity_category);
-              analytics().logEvent(GAME_DETAILS_OPENED, { game_id: articleData[0]?.id, game_category_id: articleData[0]?.activity_category });
+              const eventGameCatData= {'name': GAME_CATEGORY_SELECTED + "_" + articleData[0]?.activity_category} 
+              const eventGameDetailsData= {'name': GAME_DETAILS_OPENED, 'params': { game_id: articleData[0]?.id, game_category_id: articleData[0]?.activity_category } }  
```

Comment:
When we log events to Firebase does it log time as well? Evrim would know or we can double check on Firebase. Otherwise we can add timestamp as an additional parameter to each event:
const eventGameDetailsData= {'name': GAME_DETAILS_OPENED, 'params': { game_id: articleData[0]?.id, game_category_id: articleData[0]?.activity_category, timestamp: new Date() } }

Reply:
when user is online it will not take long time directly log with firebase and when user is offline store to realm and when back to online all stored event sync with firebase. 
If you want to add additional parameter params we can add that.